provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -2290,12 +2265,73 @@ func vpcCniProperties(kubeconfig bool) map[string]schema.PropertySpec {
 			Description: "Pass privilege to containers securityContext. This is required when SELinux is enabled. " +
 				"This value will not be passed to the CNI config by default",
 		},
+		"configurationValues": {
+			TypeSpec: schema.TypeSpec{
+				Type:                 "object",
+				AdditionalProperties: &schema.TypeSpec{Ref: "pulumi.json#/Any"},
+			},
+			Description: "Custom configuration values for the vpc-cni addon. This object must match the schema derived from " +
+				"[describe-addon-configuration]" +
+				"(https://docs.aws.amazon.com/cli/latest/reference/eks/describe-addon-configuration.html).",
+		},
+		"resolveConflictsOnCreate": {
```

Comment:
In a different PR I saw today, something had an enumeration for these, is this coming? Enumerations are nice. Thanks!

Reply:
Yeah, I referenced this in a comment above: https://github.com/pulumi/pulumi-eks/pull/1358#discussion_r1764974168

There's some cleanup left to align Cory's and my PR once they're merged